src/pages/account/organization/benefits/Banner.component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I usually move page components to subfolders because it may make the root page folder clean and tidy.

```shell
benefits/
|-- index.js
|-- icons/
  |-- decorator-lines.svg
  |-- banner/
    | -- index.js
    | -- Banner.component.js
    | -- banner.styled.js
```